ramls/items-lost-requiring-actual-costs.raml

```diff
@@ -0,0 +1,26 @@
+#%RAML 1.0
+title: Items Lost Requiring Actual Costs
+version: v0.2
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: API for fetching items lost requiring actual cost
+    content: <b>API for items lost requiring actual cost generation</b>
+
+types:
+  loan: !include loan.json
+  items-lost-requiring-actual-costs: !include items-lost-requiring-actual-costs.json
+
+traits:
+  language: !include raml-util/traits/language.raml
+
+resourceTypes:
+  collection-get: !include raml-util/rtypes/collection-get.raml
+
+/inventory-reports:
```

Comment:
I'd not use `inventory-reports` here. Is it fine to use `circulation` here? So the whole path is `/circulation/items-lost-requiring-actual-costs`